src/components/TrackNewExercise.tsx

```diff
@@ -61,6 +63,17 @@ export function TrackNewExercise(props: TrackNewExerciseProps): JSX.Element {
 
   const prepopulate = props.location.state
 
+  const formatMarkdown = (markdown: string | undefined): string | undefined => {
```

Comment:
Given that this doesn't seem to use _any_ local state, you probably want to make this a full-fledged, top-level `function`

Reply:
👍